cmd/mountlib/rc.go

```diff
@@ -4,27 +4,39 @@ import (
 	"context"
 	"log"
 	"sort"
+	"time"
 
 	"github.com/pkg/errors"
 	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/rc"
 )
 
+// MountInfo defines the configuration for a mount
+type MountInfo struct {
+	unmountFn  UnmountFn
+	MountPoint string    `json:"MountPoint"`
+	MountedOn  time.Time `json:"MountedOn"`
+	Fs         string    `json:"Fs"`
+}
+
 var (
 	// Mount functions available
 	mountFns map[string]MountFn
-	// Map of mounted path => unmount function
-	unmountFns map[string]UnmountFn
+	// Map of mounted path => MountInfo
+	liveMounts map[string]MountInfo
 )
 
+func init() {
+
+	mountFns = make(map[string]MountFn)
```

Comment:
If you want to initialise them to an empty `map` then change the var definition to

```go
    // Mount functions available
￼    mountFns = map[string]MountFn{}
    // Map of mounted path => MountInfo
￼    liveMounts = map[string]MountInfo{}
```

Reply:
The mutex should be used only for modifying the map right?